--- ./Makefile	2025-10-10 09:30:10.000000000 -0700
+++ ./Makefile	2026-01-03 13:30:36.747340088 -0800
@@ -4,7 +4,7 @@
 PREFIX =	/usr
 endif
 BINDIR = 	$(DESTDIR)/$(PREFIX)/bin
-MANDIR =	$(DESTDIR)/$(PREFIX)/man
+MANDIR =	$(DESTDIR)/$(PREFIX)/share/man
 TMP ?= /tmp
 
 PROG =	ksh
@@ -22,8 +22,9 @@
 CDIAGFLAGS+=	-Wshadow
 CDIAGFLAGS+=	-Wdeclaration-after-statement
 
-CFLAGS +=	$(CDIAGFLAGS) `getconf LFS_CFLAGS` -DEMACS -DVI `pkg-config --cflags ncurses`
-LDADD +=	-lbsd `pkg-config --libs ncurses`
+CFLAGS +=	$(CDIAGFLAGS) `getconf LFS_CFLAGS` -DEMACS -DVI `pkg-config --cflags ncursesw`
+CFLAGS +=	-fPIE -pie
+LDADD +=	-lbsd `pkg-config --libs ncursesw`
 
 $(PROG): $(OBJS)
 	$(CC) $(OBJS) $(LDADD) $(CFLAGS) \
